gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/NonLocalRegionEntry.java

```diff
@@ -46,7 +47,7 @@
 import com.gemstone.gemfire.internal.offheap.annotations.Released;
 import com.gemstone.gemfire.internal.shared.Version;
 
-public class NonLocalRegionEntry implements RegionEntry, VersionStamp {
+public class NonLocalRegionEntry implements RegionEntry, VersionStamp, DataSerializable {
```

Comment:
Yes will remove as discussed. It was for persisting NonLocalRegionEntry which was part of other design.